src/Compilers/Test/Utilities/VisualBasic/BasicTestBase.vb

```diff
@@ -869,10 +863,15 @@ Public MustInherit Class BasicTestBase
 
         Dim fileName = "a.vb"
         Dim syntaxTree = Parse(source, fileName, parseOptions)
-        Dim defaultRefs = If(useLatestFramework, LatestVbReferences, DefaultVbReferences)
-        Dim allReferences = defaultRefs.Concat({ValueTupleRef, SystemRuntimeFacadeRef})
+        Dim allReferences As IEnumerable(Of MetadataReference) = Nothing
+        If useLatestFramework Then
```

Comment:
>useLatestFramework [](start = 11, length = 18)

Consider moving into `.Add()` call.

Reply:
Will do on the next PR here.

---
In reply to: [192157523](https://github.com/dotnet/roslyn/pull/25130#discussion_r192157523) [](ancestors = 192157523)